spec.html

```diff
@@ -22544,16 +22544,15 @@
 
       <!-- es6num="19.1.2.2" -->
       <emu-clause id="sec-object.create">
-        <h1>Object.create ( _O_ [ , _Properties_ ] )</h1>
+        <h1>Object.create ( _O_, _Properties_ )</h1>
```

Comment:
This convention is already used in other places (Error constructor, Number.prototype.toExponential, String constructor, String.prototype.slice, etc.).